swagger.yml

```diff
@@ -891,42 +891,63 @@ components:
     Alert:
       title: Alert
       type: object
+      x-examples:
+        example-1:
+          AlertAffectedTechnology: null
+          AlertCategory: AGENT
+          AlertCode: NO_DATA
+          AlertSeverity: CRITICAL
+          AlertStatus: NEW
+          Date: '2020-07-08T06:15:24.391Z'
+          Description: No data received from the host publicitate-36d06ca83eafa454423d2097f4965517 in the last 0 days
+          Hostname: publicitate-36d06ca83eafa454423d2097f4965517
+          OtherInfo:
+            Hostname: publicitate-36d06ca83eafa454423d2097f4965517
```

Comment:
x-examples???

Reply:
with Stoplight it is not visible. removed manually

Reply:
I couldn't remove this tag. any idea ? @cirelli94 @amreo ??

Reply:
I can't see it either with stoplight!
We should just remove the example from the schema and place examples only in api?

Reply:
i'll leave it there.. it is not an error. what do you think ?

Reply:
x-examples is a error because it isn't specified in openapi v3.0 (https://swagger.io/specification/), The standard key is examples

Reply:
Try to rename it manually

Reply:
when I try to rename I get this error:

Structural error at components.schemas.Alert
should NOT have additional properties
additionalProperty: examples

Reply:
x-examples deleted